components/support/nimbus-fml/src/parser.rs

```diff
@@ -69,6 +70,130 @@ pub(crate) struct ManifestFrontEnd {
     channels: Vec<String>,
 }
 
+impl ManifestFrontEnd {
```

Comment:
I refactored this file a bit to unclutter the `Parser::new` function, happy to take suggestions/feedback!